03/app.test.js

```diff
@@ -1 +1,25 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+it('return 1 when the number to be drawn is between 1 and 1', () => {
+    const min = 1;
+    const max = 1;
+    const result = randomNumber(min, max);
+
+    expect(result).toBe(1);
+});
+
+it('throw exception when argument is not a number', () => {
+    function getRandomNumber() {
+        randomNumber('1', 5);
+    }
+
+    expect(getRandomNumber).toThrow()
```

Comment:
Należy wskazać jaki to błąd (jaką ma treść tj. `toThrow('message')` - wtedy dopiero będziemy pewni, że działa to jak należy